web/lib/components/flashcards/PageAllSets.jsx

```diff
@@ -157,12 +163,14 @@ export function PageAllSets({ page }) {
       </div>
 
       {/* Navigators */}
-      <PageNavigation
-        currentPage={currentPage}
-        onUpdateCurrentPage={setCurrentPage}
-        isNextDisabled={flashcardSets.length < PER_PAGE}
-        isPrevDisabled={currentPage === 0}
-      />
+      {total > PER_PAGE && (
+        <PageNavigation
+          currentPage={currentPage}
+          onUpdateCurrentPage={setCurrentPage}
```

Comment:
Is `currentPage` used somewhere else in this component? If not we don't need to define it as a state and `onUpdateCurrentPage` etc. can be removed.

Reply:
can remove `currentPage` though we would need to reload of each `?page={number}`, while keep `currentPage` will only re-render `Pagination`

Reply:
Prolly it's not that bad, because we have react-query cached the result already 🤔 URL becomes single source of truth, which is good.